guides/user/canary/best-practices/index.md

```diff
@@ -16,6 +16,111 @@ critical metric fails, but the rest pass, the group gets a passing score overall
+This gives you 3 canary runs, each 1 hour long.
+
+## Carefully choose your thresholds
+
+You need to configure two thresholds for a canary analysis:
+
+* marginal
+
+  If a canary run has a score below than this threshold, then the whole canary
+  fails.
+
+* pass
+
+  The last canary run of the analysis must score higher than this threshold for
+  the whole analysis to be considered successful. Otherwise it fails.
+
+These thresholds are very important for the analysis to give an accurate result.
+You need to experiment with them, in the context of your own application, its
+traffic and its metrics.
+
+Good starting points:
+
+* marginal threshold of 75
+* pass threshold of 95
+
+## Carefully choose the metrics to analyze
+
+You can get started with a single metric, but in the long run, your canaries
+will use several.
+
+Use a variety of metrics that reflect different aspects of the health of your
+application. Use these three out of the four "golden signals," as defined in
+the [SRE Book](https://landing.google.com/sre/book/chapters/monitoring-distributed-systems.html):
+
+* latency
+* errors
+* saturation
+
+If you consider some other specific metrics critical, place them in their own
+group in the canary configuration. That allows you to fail the whole canary
+analysis if there is a problem with one of those specific metrics.
+
+## Create a standard, resusable canary config
+
+Configuring a canary is difficult, and not every developer in your organization
+will be able to do so. Also, if you let all the teams in the org manage their
+own configs, you will likely end up with too many configs, too many metrics,
+nobody will know what is happening, and people will be afraid to change
+anything.
+
+For these reasons, it's a good idea to curate a config that all the teams can
+reuse.
+
+## Use retrospective analysis to make debugging faster
+
+It takes a long time to configure a canary analysis. It can take a long time to
+debug it too, partly because with [a long-running canary
+analysis](#run-the-canary-for-a-long-enough-time) you have to wait a long time
+for the analysis to finish before you can refine it.
+
+Fortunately, a Canary Analysis stage can be configured to use a [retrospective
+analysis](/guides/user/canary/stage/#real-time-versus-retrospective-analysis)
+instead of a real-time analysis. This analysis is based on past monitoring data,
+without having to wait for the data points to be generated. With this mode, you
+can iterate more quickly on the development of the canary configuration.
+
+## Compare equivalent deployments
+
+To compare the metrics between baseline and canary, Kayenta needs the exact same
+metrics for both. This means that metrics should have the same labels. Problems
+can arise if the metrics are labeled with their respective instance names.
+
+If you see that Kayenta is not actually comparing the metrics, confirm that the
+queries that it does to your monitoring system return metrics with the same
```

Comment:
s/does to your/executes against your

The easiest way to view the actual expanded queries is to navigate to the `Metrics` link under `Source` in the Reports view.